tests/unit/backend/wmg/fixtures/test_primary_filters_local.py

```diff
@@ -0,0 +1,42 @@
+test_organism_terms = [
+    {"organism_ontology_term_id_0": "Mus musculus"},
```

Comment:
If possible I would use the correct ontology ids
Many of them are predefined in `tests/unit/backend/wmg/fixtures/test_primary_filters.py`